torch/_dynamo/convert_frame.py

```diff
@@ -456,6 +456,7 @@ def transform(instructions, code_options):
         if hooks.guard_export_fn is not None:
             hooks.guard_export_fn(output.guards)
 
+        output.clear_live_user_objs()
```

Comment:
is there a reason this cant be here https://github.com/pytorch/pytorch/blob/40d6f2a02027023216607adb892d3b9c7493904c/torch/_dynamo/output_graph.py#L738 ?

Reply:
Good point, done. Thought it was only cleaned once the code was invalidated by the CleanupManager